libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -5,16 +5,10 @@ import {
   ContainerClient,
 } from "@azure/storage-blob";
 import mime from "mime-types";
-import {
-  IBucketClient,
-  ObjectMetadata,
-  BucketPutOptions,
-  BucketDeleteOptions,
-  BucketSignedUrlOptions,
-} from "../cloud";
+import { IBucketClient, BucketPutOptions, BucketDeleteOptions } from "../cloud";
 import { Json } from "../std";
 
-export class BucketClient implements IBucketClient {
+export class BucketClient implements Partial<IBucketClient> {
```

Comment:
It feels sketchy to use partial here. I think an explicit throw from each non implemented method is more readable